File: benchmark/results/java-invocation-inline/run-005/b-baseline/output/com/metaengine/demo/analytics/value_objects/EventPayload.java
package com.metaengine.demo.analytics.value_objects;

import java.time.Instant;

/** EventPayload value object — payload data carried by an Event. */
public record EventPayload(
    String id,
    Instant createdAt,
    Instant updatedAt,
    String name,
    String description
) {}

File: benchmark/results/python-multilang/run-001/b-baseline/output/metaengine_demo/analytics/value_objects/event_payload.py
"""EventPayload value object for the analytics domain."""
from dataclasses import dataclass
from datetime import datetime


@dataclass(frozen=True)
class EventPayload:
    """Immutable EventPayload value object."""

    id: str
    createdAt: datetime
    updatedAt: datetime
    name: str
    description: str
